packages/libs/client-utils/src/marmalade/create-token.ts

```diff
@@ -139,7 +139,7 @@ const generatePolicyTransactionData = (
       );
   }
 
-  if (policyConfig?.guarded && !policyConfig?.updatableURI) {
+  if (policyConfig?.guarded && policyConfig?.nonUpdatableURI) {
```

Comment:
With addition of nonUpdatableURI, we now don't have to enforce usage of guard policy with the registration of a failure guard token uri is non-updatable, because the existence of `non-updatable-uri-policy` will do so. 

One suggestion I'd make is enforcing uri guard registration for updatable uri's , This way we'll restrict the option of not using guard policy to handle the update-uri logic, but we be conservative and prevent an updatable uri token being created without an uri guard. Open for hearing different opinions

Reply:
You're right! I have updated the code to reflect this